plugins/jobs/web_client/views/timeChartConfig.js

```diff
@@ -270,7 +270,7 @@ export default {
     'legends': [
         {
             'fill': 'timing',
```

Comment:
I see what you mean. The legend of on the line chart is an indicator of what phases has been selected by the filter.